src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -647,14 +650,11 @@ def apply_sbp_treatment_ramp(
             exposure_pipeline = self.sbp
 
         overcome_initial_prescription_inertia = pop_visitors[
-            self.initial_prescription_inertia_propensity[pop_visitors.index]
```

Comment:
I'd rename this variable to `overcome_prescription_inertia` (i.e. drop initial). 

Also I'd need to get this data from the state table like:
```
sbp_prescription_inertia_propensity = pop_visitors["sbp_prescription_inertia_propensity"]
overcome_prescription_inertia = ...  # same as you have done
```